fence/README.md

```diff
@@ -0,0 +1,155 @@
+# Network Fencing
+
+## Terminology
+
+| Term     | Definition                                                                            |
+| -------- | ------------------------------------------------------------------------------------- |
+| CO       | Container Orchestration system that communicates with plugins using CSI service RPCs. |
+| SP       | Storage Provider, the vendor of a CSI plugin implementation.                          |
+| DR       | Disaster Recovery.                                                                    |
+| RPC      | [Remote Procedure Call](https://en.wikipedia.org/wiki/Remote_procedure_call).         |
+
+## Objective
+
+Define a standard that will enable storage vendors (SP) to
+perform node level fencing using corresponding CIDR blocks.
+
+### Goals in MVP
+
+The new extension will define a procedure that allows SP to blocklist
+any client using corresponding CIDR blocks.
+
+If multiple COs are accessing a shared resource provided by the SP, and a node
+failure takes place, this procedure will allow the SP to blocklist any node of CO,
+and thus preventing further access and corruption of data.
+
+### Non-Goals in MVP
+
+* Application level fencing.
```

Comment:
I don't understand what is meant with this. Maybe expand a little more on that, or link to a description of "application level fencing".

Reply:
added more details on the same